src/utilities/getValue.ts

```diff
@@ -0,0 +1,13 @@
+import { TransformedToken } from "style-dictionary/types";
+
+/**
+ * getValue
+ * @description Returns the value of the design token, either token.value or token.$value
+ * @param token StyleDictionary.DesignToken
+ * @returns nested json three
+ */
+export const getValue = (
+  token: TransformedToken | Record<string, unknown>
+): unknown => {
```

Comment:
@AndreBarr maybe we could provide a type `getValue<string>(token)` so that we can define which value is returned? https://www.typescriptlang.org/docs/handbook/2/generics.html

Reply:
Updated